libcore/src/androidTest/java/com/mapbox/android/core/CoreSDKVersionTest.java

```diff
@@ -0,0 +1,44 @@
+package com.mapbox.android.core;
+
+import android.content.Context;
+import android.content.res.AssetManager;
+import android.support.test.InstrumentationRegistry;
+import android.util.Log;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import static org.junit.Assert.fail;
+
+public class CoreSDKVersionTest {
+
+  private static final String SECOND_LINE_FORMAT = "v%d";
+  private static final String SDK_VERSIONS_FOLDER = "sdk_versions";
+  private static final String LOG_TAG = "CoreSDKVersionTest";
+
+  @Test
+  public void testPersistedCoreSDKInfo() {
+    Context context = InstrumentationRegistry.getTargetContext();
+    AssetManager assetManager = context.getAssets();
+    InputStream is = null;
+
+    try {
+      is = assetManager.open(SDK_VERSIONS_FOLDER + File.separator + context.getPackageName()
+        .replace(".test", ""));
```

Comment:
possibly more readable with `context.getPackageName().replace(".test", "")` extracted to a variable